src/components/Login.jsx

```diff
@@ -0,0 +1,115 @@
+import { FaEye, FaEyeSlash } from "react-icons/fa";
+import { useReducer } from "react";
+import { Link } from "react-router-dom";
+import { LoginIllustration } from "components";
+export function Login({ formik }) {
+  const [showPassword, toggleShowPassword] = useReducer(
+    (state) => !state,
+    false
+  );
+  return (
+    <div className="min-h-screen bg-primary p-4 md:p-16">
+      <div className="rounded-2xl bg-light-100 dark:bg-dark-100 lg:flex">
+        <div className=" md:rounded-r-0 rounded-2xl bg-light-200 p-8 dark:bg-dark-200 md:rounded-l-2xl xl:max-w-screen-sm">
+          {/* <div className="py-12 bg-light-200 dark:bg-dark-200 flex justify-center lg:justify-start lg:px-12">
+            <div className="cursor-pointer flex items-center">
+              <div className="text-2xl text-primary tracking-wide ml-2 font-semibold">
+                Social Gully
+              </div>
+            </div>
+          </div> */}
```

Comment:
i think you should remove this

Reply:
Thanks for pointing it out, I have removed it in the latest commit